starter-repos/app-template/.gitlab-ci.yml

```diff
@@ -16,6 +16,11 @@
 include:
 - project: 'platform-admins/shared-ci-cd'
   file: 'ci/go-docker.yaml'
+- project: 'platform-admins/shared-ci-cd'
+  file: 'skaffold/build.yaml'
```

Comment:
`shared-ci-cd/skaffold/build.yaml` uses `skaffold build -p ci`, but we don't have ci profile in skaffold.yaml.

Reply:
good catch. added a ci profile.